packages/ui/build/css/themes/light.css

```diff
@@ -1,59 +1,61 @@
-  --brand-600: 153.2deg 49.7% 33.5%;
-  --brand-500: 148.7deg 42.7% 69.2%;
-  --brand-400: 149.1deg 59.3% 88.4%;
-  --brand-300: 148.6deg 77.8% 94.7%;
-  --brand-200: 156deg 71.4% 98.6%;
+  --brand-600: 156.5deg 86.5% 26.1%;
+  --brand-500: 155.3deg 78.4% 40%;
+  --brand-400: 151.3deg 66.9% 66.9%;
+  --brand-300: 147.5deg 72% 80.4%;
+  --brand-200: 147.6deg 72.5% 90%;
   --warning-default: 38.9deg 100% 57.1%;
-  --warning-600: 30.2deg 100% 33.9%;
+  --warning-600: 30.3deg 80.3% 47.8%;
   --warning-500: 36.3deg 85.7% 67.1%;
   --warning-400: 41.9deg 100% 81.8%;
   --warning-300: 44.3deg 100% 91.8%;
-  --warning-200: 40deg 60% 99%;
+  --warning-200: 40deg 81.8% 97.8%;
   --destructive-default: 10.2deg 77.9% 53.9%;
   --destructive-600: 9.9deg 82% 43.5%;
   --destructive-500: 10.4deg 77.1% 79.4%;
   --destructive-400: 7.1deg 91.3% 91%;
   --destructive-300: 7.1deg 100% 96.7%;
   --destructive-200: 0deg 100% 99.4%;
-  --background-alternative-default: 0deg 0% 100%;
+  --border-stronger: 205.7deg 6.3% 56.1%;
+  --border-strong: 206.7deg 10.3% 82.9%;
+  --border-default: 216deg 11.1% 91.2%;
+  --background-muted: 210deg 12.5% 96.9%;
+  --background-surface-400: 220deg 5.9% 90%;
+  --background-surface-300: 210deg 5.6% 92.9%;
+  --background-surface-200: 0deg 0% 95.3%;
+  --background-surface-100: 0deg 0% 98.8%;
   --background-alternative-200: 0deg 0% 100%;
+  --background-200: 210deg 33.3% 98.8%;
   --foreground-contrast: 210deg 25% 98.4%;
+  --foreground-muted: 0deg 0% 69.8%;
+  --foreground-lighter: 0deg 0% 43.9%;
+  --foreground-light: 0deg 0% 32.2%;
   --border-button-hover: var(--colors-slate-light-700);
   --border-button-default: var(--colors-slate-light-600);
-  --border-stronger: var(--colors-gray-light-800);
-  --border-strong: var(--colors-gray-light-700);
   --border-alternative: var(--colors-slate-light-500);
   --border-control: var(--colors-slate-light-600);
   --border-overlay: var(--colors-slate-light-500);
   --border-secondary: var(--colors-slate-light-400);
   --border-muted: var(--colors-slate-light-400);
-  --border-default: var(--colors-slate-light-500);
   --background-button-default: var(--colors-slate-light-100);
-  --background-muted: var(--colors-slate-light-400);
   --background-overlay-hover: var(--colors-slate-light-300);
-  --background-overlay-default: var(--colors-slate-light-100);
-  --background-surface-300: var(--colors-slate-light-400);
-  --background-surface-200: var(--colors-slate-light-300);
-  --background-surface-100: var(--colors-slate-light-100);
+  --background-overlay-default: var(--colors-slate-light-300);
   --background-control: var(--colors-slate-light-300);
   --background-selection: var(--colors-gray-light-400);
+  --background-alternative-default: var(--colors-slate-light-100);
   --background-default: var(--colors-slate-light-200);
-  --foreground-muted: var(--colors-slate-light-900);
-  --foreground-lighter: var(--colors-slate-light-1000);
-  --foreground-light: var(--colors-slate-light-1100);
```

Comment:
Privacy toast looks a bit weird now and the hover on the items in the Command isn't distinguishable from the bg.

<img width="1215" alt="Screenshot 2024-01-25 at 15 33 40" src="https://github.com/supabase/supabase/assets/25671831/881034af-208c-4c43-8f96-3def7a501967">